helm/chart/doc.yaml

```diff
@@ -134,6 +134,7 @@
 ## @param   lifecycleOperator.manager.env.otelCollectorUrl Sets the URL for the open telemetry collector
 ## @param   lifecycleOperator.manager.env.functionRunnerImage specify image for deno task runtime
 ## @param   lifecycleOperator.manager.env.pythonRunnerImage specify image for python task runtime
+## @param   lifecycleOperator.manager.env.schedulingGatesEnabled enables the scheduling gates in lifecycle-operator
```

Comment:
I would add more info in this, for instance that scheduling gates requires k8s 1.27.0 to use it or 1.26.0 enabling the alpha feature